.github/ISSUE_TEMPLATE/PROBLEM_REPORT.yml

```diff
@@ -1,5 +1,5 @@
 name: Report a Problem
-description: File an problem report
+description: Something does not work well, is too hard to do or missing at all? Please report a Problem.
```

Comment:
Have you found something that does not work well, is too hard to do or is missing altogether? Please create a Problem Report.

Reply:
Seems better to me, I've updated the file.